install_config/persistent_storage/persistent_storage_cinder.adoc

```diff
@@ -190,3 +190,27 @@ spec:
 <7> Specify an
 xref:../../install_config/persistent_storage/pod_security_context.adoc#fsgroup[`*fsGroup*`]
 for the pods.
+
+[[volume-limit-cinder]]
+=== Cinder volume limit
+
+By default, a maximum of 256 Cinder volumes can be attached to each node in a cluster.
+To change this limit:
+
+. Set the `KUBE_MAX_PD_VOLS` environment variable to an integer. For example, in `/etc/origin/master/master.env`:
++
+----
+KUBE_MAX_PD_VOLS=26
+----
+
+. From a command line, restart the API service:
++
+----
+$ master-restart api
```

Comment:
A quick check of 3.11 docs shows `master-restart api` and `master-restart controllers` (below) to be executed from root. For example: https://docs.openshift.com/container-platform/3.11/install_config/configuring_vsphere.html#vsphere-applying-configuration-changes

So maybe `$` -> `#` ?

Reply:
@bobfuru Fair. 

@chao007 Could you comment on this? The change would warrant a QE re-check if it happened, anyway. :)

Reply:
Yes. We should update to `#` here

Reply:
@chao007 Thanks!